packages/relay/src/relay.ts

```diff
@@ -59,6 +62,10 @@ export interface Relay {
   setSubscriptionHandler(handler: SubscriptionRequestHandler): this;
 }
 
+type LitePubRelayFollower = {
+  actor: string;
+  state: string;
+};
```

Comment:
I think the `actor` field should be `unknown` so that it has a JSON object?

```suggestion
interface LitePubRelayFollower {
  readonly actor: unknown;
  readonly state: string;
};
```